.flake8

```diff
@@ -7,6 +7,9 @@ docstring-convention=google
 # E203 and W503 don't interact well with black
 ignore = D205,D212,D415,E203,W503
 
+exclude = 
+    xrpl/openapi-codegen/*
```

Comment:
I wouldn't exclude the folder, if there's something wrong with the typing then the codegen probably needs to be fixed

Reply:
Same here, will fix the pipeline